supervisor/addons/addon.py

```diff
@@ -453,7 +453,7 @@ async def watchdog_application(self) -> bool:
         application = RE_WATCHDOG.match(url)
 
         # extract arguments
-        t_port = application.group("t_port")
+        t_port = int(application.group("t_port"))
```

Comment:
Shouldn't this be on L469 instead?

Reply:
Why? we extract the port number here and we should make sure it is a number at any point. This is as close to the source as possible